### PR TITLE
Add log messages and fix multi-arch build

### PR DESCRIPTION
**Pull request by bn222**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -153,16 +153,14 @@ images-build: ## Build all container images necessary to run the whole operator
...
-	buildah build --authfile /root/config.json --manifest tmp-manifest --arch arm64 -v $(GO_CONTAINER_CACHE):/go:z -f Dockerfile.rhel -t $(DPU_OPERATOR_IMAGE)
-	buildah manifest rm tmp-manifest2
-	buildah manifest create tmp-manifest2 || true
-	buildah build --authfile /root/config.json --manifest tmp-manifest2 --arch amd64 -v $(GO_CONTAINER_CACHE):/go:z -f Dockerfile.daemon.rhel -t $(DPU_DAEMON_IMAGE)
-	buildah build --authfile /root/config.json --manifest tmp-manifest2 --arch arm64 -v $(GO_CONTAINER_CACHE):/go:z -f Dockerfile.daemon.rhel -t $(DPU_DAEMON_IMAGE)
-	buildah manifest push --all tmp-manifest docker://$(DPU_OPERATOR_IMAGE)
-	buildah manifest push --all tmp-manifest2 docker://$(DPU_DAEMON_IMAGE)
+	buildah manifest rm $(DPU_OPERATOR_IMAGE)-manifest || true
+	buildah manifest create $(DPU_OPERATOR_IMAGE)-manifest
+	buildah build --authfile /root/config.json --manifest $(DPU_OPERATOR_IMAGE)-manifest --platform linux/amd64,linux/arm64 -v $(GO_CONTAINER_CACHE):/go:z -f Dockerfile.rhel -t $(DPU_OPERATOR_IMAGE)
```
Comment (SalDaniele) [inline Makefile:158]: Looks like the cross-compilation is working now with this change

File: daemon/hostdaemon.go [hunk 2]
```diff
@@ -99,7 +99,7 @@ func (d *HostDaemon) connectWithRetry() error {
 	// retries)
 	retryPolicy := `{
 		"methodConfig": [{
-		  "waitForReady": false,
+		  "waitForReady": true,
```
Comment (wizhaoredhat) [inline daemon/hostdaemon.go:102]: Left over debug?

File: daemon/hostdaemon.go [hunk 3]
```diff
@@ -114,29 +114,32 @@ func (d *HostDaemon) connectWithRetry() error {
...
 	if err != nil {
 		return nil, fmt.Errorf("SRIOV manager failed in add handler: %v", err)
 	}
+	d.log.Info("addHandler d.sm.CmdAdd succeeded")
 	pf := 0
 	vf := req.CNIConf.VFID
 	mac := req.CNIConf.OrigVfState.EffectiveMAC
 	d.log.Info("addHandler", "CNIConf", req.CNIConf)
-	vlan := *req.CNIConf.Vlan
+	// TODO: fix setting Vlan based on network definition in CR
```
Comment (wizhaoredhat) [inline daemon/hostdaemon.go:134]: Talking to Balazs offline about this weirdness.

**Pull request merged**